File: Semana 3/Tarea encargada de funciones (cuaderno)/python/Ejercicio9.py
def ejercicio9(a, b):
    return b, a  # En Python se intercambian con múltiple asignación

x, y = 10, 20
print(f"Antes: x={x}, y={y}")
x, y = ejercicio9(x, y)
print(f"Despues: x={x}, y={y}")

File: Semana 3/Tarea encargada de funciones (cuaderno)/C++/Ejercicio9.cpp
#include <iostream>
using namespace std;

void Ejercicio9(int &a, int &b) {
    int temp = a;
    a = b;
    b = temp;
}

int main() {
    int x = 10, y = 20;
    cout << "Antes: x=" << x << ", y=" << y << endl;
    Ejercicio9(x, y);
    cout << "Despues: x=" << x << ", y=" << y << endl;
    return 0;
}
